recipe/meta.yaml

```diff
@@ -19,16 +19,20 @@ outputs:
       host:
         - python >=3.8
         - pip
+        - setuptools >=64
+        - setuptools-scm >=7
       run:
         - python >=3.8
         - cartopy >=0.20
-        - cf-units >=3
-        - cftime >=1.5.0 
+        - cf-units ==3.0.1
+        - cftime ==1.5.1.1
```

Comment:
These pins should reflect [those in Iris GitHub](https://github.com/SciTools/iris/blob/aa2dca38d3d462a5ca9133aa313c3f5285564bdf/requirements/ci/py310.yml#L15-L16).

(Sorry I didn't understand what you meant when we were talking yesterday).

Reply:
This reflects the state of the pins in the main branch of the feedstock as introduced in #83. I'm not aware of any reason for these being safe to remove now so I figured it would be safer to leave them in

Reply:
We've now gone for the pins I suggested after we remembered why #82 set them in the first place.